File: frontend/src/featureFlagDefaults.js
/**
 * Local default values for feature flags evaluated by the frontend.
 *
 * These are the fallbacks used before the sidecar's flag bag arrives (and
 * permanently in Privacy Lockdown, which suppresses the server fetch).
 * When adding a new flag, add it here AND to
 * backend/feature_flag_defaults.py so first-launch behavior is
 * deterministic in both processes.
 *
 * Flag names should be lowercase-snake-case.
 */
export const FLAG_DEFAULTS = {
  // Example: flow_v3_ui: false,
}

File: backend/feature_flag_defaults.py
"""
Local default values for feature flags evaluated by the sidecar.

These are the fallbacks used before the first ``/api/feature-flags``
fetch completes (and permanently under ``DO_NOT_TRACK=1``, which
suppresses the fetch). When adding a new flag, add it here AND to
``frontend/src/featureFlagDefaults.js`` so first-launch behavior is
deterministic in both processes.

Flag names should be lowercase-snake-case.
"""
from typing import Any, Dict

FLAG_DEFAULTS: Dict[str, Any] = {
    # Example: "flow_v3_ui": False,
}
